pkg/pointerdb/pointerdb.go

```diff
@@ -53,11 +59,41 @@ func (s *Server) validateAuth(APIKey []byte) error {
 	return nil
 }
 
+func (s *Server) validateSegment(req *pb.PutRequest) error {
+	min := s.config.MinInlineSegmentSize
+	max := s.config.MaxInlineSegmentSize
+	inlineSize := len(req.GetPointer().InlineSegment)
+	pointer := req.GetPointer()
+	remote := pointer.Remote
+
+	fmt.Printf("Pointer is %+v\n", pointer)
+	fmt.Printf("Remote is %+v\n", remote)
```

Comment:
can you change these to debug logs?

Reply:
they weren't supposed to be in there, I just removed them instead.